filesystem/iso9660.ksy

```diff
@@ -7,206 +7,84 @@ meta:
+    io: _root._io
+    pos: _root.sector_size * 0x10
+    type: volume_descriptor
+    size: sector_size
+    repeat: until
+    repeat-until: _.type == volume_type::volume_descriptor_set_terminator
+enums:
+  volume_type:
+    0x00: boot_record_volume_descriptor
+    0x01: primary_volume_descriptor
+    0x02: supplementary_volume_descriptor
+    0x03: volume_partition_descriptor
+    0xff: volume_descriptor_set_terminator
 types:
-  vol_desc:
-    seq:
-      - id: type
-        type: u1
-      - id: magic
-        contents: "CD001"
-      - id: version
-        type: u1
-      - id: vol_desc_boot_record
-        type: vol_desc_boot_record
-        if: type == 0
-      - id: vol_desc_primary
-        type: vol_desc_primary
-        if: type == 1
-  vol_desc_boot_record:
+  u2bi:
+    doc-ref: ecma-119 7.2.3
     seq:
-      - id: boot_system_id
-        type: str
-        size: 32
-        encoding: UTF-8
-      - id: boot_id
-        type: str
-        size: 32
-        encoding: UTF-8
-  vol_desc_primary:
-    doc-ref: 'http://wiki.osdev.org/ISO_9660#The_Primary_Volume_Descriptor'
+      - id: le
+        type: u2le
+      - id: be
+        type: u2be
+  u4bi:
+    doc-ref: ecma-119 7.3.3
     seq:
-      - id: unused1
-        contents: [0]
-      - id: system_id
-        type: str
-        size: 32
-        encoding: UTF-8
-      - id: volume_id
-        type: str
-        size: 32
-        encoding: UTF-8
-      - id: unused2
-        contents: [0, 0, 0, 0, 0, 0, 0, 0]
-      - id: vol_space_size
-        type: u4bi
-      - id: unused3
-        contents: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-      - id: vol_set_size
-        type: u2bi
-      - id: vol_seq_num
-        type: u2bi
-      - id: logical_block_size
-        type: u2bi
-      - id: path_table_size
-        type: u4bi
-      - id: lba_path_table_le
-        type: u4le
-      - id: lba_opt_path_table_le
+      - id: le
         type: u4le
-      - id: lba_path_table_be
-        type: u4be
-      - id: lba_opt_path_table_be
+      - id: be
         type: u4be
-      - id: root_dir
-        type: dir_entry
-        size: 34
-      - id: vol_set_id
-        type: str
-        size: 128
-        encoding: UTF-8
-      - id: publisher_id
-        type: str
-        size: 128
-        encoding: UTF-8
-      - id: data_preparer_id
+  datetime_long:
+    doc-ref: ecma-119 8.4.26.1
+    seq:
+      - id: year
         type: str
-        size: 128
-        encoding: UTF-8
-      - id: application_id
+        size: 0x4
+      - id: month
         type: str
-        size: 128
-        encoding: UTF-8
-      - id: copyright_file_id
+        size: 0x2
+      - id: day
         type: str
-        size: 38
-        encoding: UTF-8
-      - id: abstract_file_id
+        size: 0x2
+      - id: hour
         type: str
-        size: 36
-        encoding: UTF-8
-      - id: bibliographic_file_id
+        size: 0x2
+      - id: minute
```

Comment:
in the previous occurence it was `min`

Reply:
My variables are the same as in the document, but I follow multiple documents.
So the naming style does sometimes change.

If I rename things the validation against the original spec would get much harder.

Reply:
`-orig-id: `

Reply:
Same discussion: https://github.com/kaitai-io/kaitai_struct_formats/pull/82#discussion_r166069793